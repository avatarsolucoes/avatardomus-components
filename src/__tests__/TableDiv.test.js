import React from 'react';
import { shallow } from 'enzyme';
import { TableDiv, DivLine, DivCell, SupLabel } from '..';

describe('TableDiv', () => {
  it('is truthy', () => {
    expect(TableDiv).toBeTruthy();
    expect(DivLine).toBeTruthy();
    expect(DivCell).toBeTruthy();
    expect(SupLabel).toBeTruthy();
  });

  it('Test TableDiv renderer whith children', () => {
    const wrapper = shallow(
      <TableDiv className="test">
        <DivLine>
          <DivCell>
            <SupLabel>teste:</SupLabel>
            teste
          </DivCell>
        </DivLine>
        <DivLine fstart>
          <DivCell>TESTE</DivCell>
        </DivLine>
        <DivLine fend>
          <DivCell>TESTE</DivCell>
          <DivCell>TESTE</DivCell>
        </DivLine>
        <DivLine fcenter>
          <DivCell width={32}>TESTE</DivCell>
          <DivCell>TESTE</DivCell>
          <DivCell>TESTE</DivCell>
        </DivLine>
        <DivLine faround>
          <DivCell>TESTE</DivCell>
          <DivCell>TESTE</DivCell>
          <DivCell>TESTE</DivCell>
          <DivCell>TESTE</DivCell>
        </DivLine>
        <DivLine fbetween>
          <DivCell>TESTE</DivCell>
          <DivCell>TESTE</DivCell>
          <DivCell>TESTE</DivCell>
          <DivCell>TESTE</DivCell>
          <DivCell>TESTE</DivCell>
        </DivLine>
      </TableDiv>,
    );
    expect(wrapper.html()).toMatchSnapshot();
  });
});
