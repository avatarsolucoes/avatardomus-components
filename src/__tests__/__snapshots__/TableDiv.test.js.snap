// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableDiv Test TableDiv renderer whith children 1`] = `"<div class=\\"table-div-main test\\"><div class=\\"table-div-line\\"><div class=\\"table-div-cell\\"><span class=\\"table-div-suplabel\\">teste:</span>teste</div></div><div class=\\"table-div-line fstart\\"><div class=\\"table-div-cell\\">TESTE</div></div><div class=\\"table-div-line fend\\"><div class=\\"table-div-cell\\">TESTE</div><div class=\\"table-div-cell\\">TESTE</div></div><div class=\\"table-div-line fcenter\\"><div class=\\"table-div-cell\\">TESTE</div><div class=\\"table-div-cell\\">TESTE</div><div class=\\"table-div-cell\\">TESTE</div></div><div class=\\"table-div-line faround\\"><div class=\\"table-div-cell\\">TESTE</div><div class=\\"table-div-cell\\">TESTE</div><div class=\\"table-div-cell\\">TESTE</div><div class=\\"table-div-cell\\">TESTE</div></div><div class=\\"table-div-line fbetween\\"><div class=\\"table-div-cell\\">TESTE</div><div class=\\"table-div-cell\\">TESTE</div><div class=\\"table-div-cell\\">TESTE</div><div class=\\"table-div-cell\\">TESTE</div><div class=\\"table-div-cell\\">TESTE</div></div></div>"`;
